backend/db/seeds.py

```diff
@@ -0,0 +1,15 @@
+def seed():
+    from . import db
+    from .models import Event, Performance, Adjudication
+
+    event = Event(event_title="test event")
+    db.session.add(event)
```

Comment:
Let's use the `Event` class instead of the `db` object. (`Event.create(event_title="test_event")`, etc)